popups/scratch-messaging/popup.css

```
@@ -270,16 +272,20 @@ a.message-type-title-text {
 }
 
 .status-container {
-  padding-top: 10px;
-  padding-bottom: 20px;
+  padding-top: 15px;
+  padding-bottom: 15px;
```

```suggestion
  padding: 15px 0;
```
I would also decrease the padding here a little too but that's not really important here.